packages/module-loader-commonjs/src/index.ts

```diff
@@ -5,7 +5,7 @@ import fetch from 'node-fetch';
 import path from 'path';
 
 // This must be a local directory to make node module resolution possible.
-const cacheDirname = path.join(__dirname, '.cache');
+const tmpDirname = path.join(__dirname, '.tmp');
```

Comment:
Why do you changed the name to tmp?

Reply:
Because `cache` is misleading, since we explicitly do not cache fetched modules (even before this fix). They are only written to disk to make them `require`able.